RESOURCES/FEATURE_FLAGS.md

```diff
@@ -57,9 +66,23 @@ These features flags are **safe for production** and have been tested.
 - SQL_VALIDATORS_BY_ENGINE [(docs)](https://superset.apache.org/docs/installation/sql-templating)
 - SQLLAB_BACKEND_PERSISTENCE
 - THUMBNAILS [(docs)](https://superset.apache.org/docs/installation/cache)
+- ENABLE_EXPLORE_DRAG_AND_DROP
+- ENABLE_DND_WITH_CLICK_UX
+- EMBEDDABLE_CHARTS
+- EMBEDDED_SUPERSET
+- ALERTS_ATTACH_REPORTS
+- ALLOW_ADHOC_SUBQUERY
+- DRUID_JOINS
+- DISABLE_LEGACY_DATASOURCE_EDITOR
+- ENFORCE_DB_ENCRYPTION_UI
 
 ## Deprecated Flags
 These features flags currently default to True and **will be removed in a future major release**. For this current release you can turn them off by setting your config to False, but it is advised to remove or set these flags in your local configuration to **True** so that you do not experience any unexpected changes in a future release.
 
 - ALLOW_DASHBOARD_DOMAIN_SHARDING
 - DISPLAY_MARKDOWN_HTML
+- FORCE_DATABASE_CONNECTIONS_SSL
+
+## In need of categorization
+- ENABLE_ADVANCED_DATA_TYPES
```

Comment:
Hey @rusackas sorry been on an assignment and missed this! this should be in development

Reply:
Perfect, thanks!